hermes/views.py

```diff
@@ -181,9 +180,137 @@ def get(self, request, *args, **kwargs):
         {title: <Title of the message>,
          topic: <kafka topic to post message to>, 
          submitter: <submitter of the message>,
-         authors: <Text full list of authors on a message>
+         authors: <Text full list of authors on a message>,
          message_text: <Text of the message to send>,
-         data: {<Unparsed json data dict>}
+         data: {
+            references: [
+                {
+                    source:
+                    citation:
+                    url:
+                },
+                ...
+            ],
+            extra_data: {
+                key1: value1,
+                key2: value2,
+                ...
+            },
+            event_id: <Nonlocalized event_id this message relates to>,
+            targets: [
+                {
+                    name: <Target name>,
+                    ra: <Target ra in decimal or sexigesimal format>,
+                    dec: <Target dec in decimal or sexigesimal format>,
+                    ra_error: <Error of ra>,
+                    dec_error: <Error of dec>,
+                    ra_error_units: <Units for ra error>,
+                    dec_error_units: <Units for dec error>,
+                    pm_ra: <RA proper motion in arcsec/year>,
+                    pm_dec: <Dec proper motion in arcsec/year>,
+                    epoch: <Epoch in MJD>,
```

Comment:
You set this to `default='J2000'` in the serializers, so this isn't MJD.
J2000 is in Julian years (an idealized year of exactly 365.25 days of 86400 SI seconds each.)
I would simply put `<Epoch of reference frame in string format)>` or something and leave it at that. In theory it can be a number of different formats, but in all likelihood nobody will ever use anything other than J2000.